ote/config.edn

```diff
@@ -15,4 +15,6 @@
        :export-base-url "http://localhost:8080"}
 
  :log {:level :debug}
- :ga {:tracking-code "UA-113014221-1"}}
+ :ga {:tracking-code "UA-113014221-1"}
+ :enabled-features "example-feature, " ;; add features to the string, split by "," to enable them in production
```

Comment:
this would be better as a set `#{:example-feature :other-feature ...}`

Reply:
I tried it. It is a string in that html page and I could not figure out how "#{:my-key}" is converted to set